### PR TITLE
Add diversity & inclusion post

### PR DESCRIPTION
**Pull request by vipulgupta2048**
Signed-off-by: Vipul Gupta (@vipulgupta2048) <vipulgupta2048@gmail.com>

### CODE REVIEW

**Review by sukanyamandal — CHANGES_REQUESTED** [review]
2nd paragraph first line - 

As per our [mission statement](https://in.pycon.org/blog/2019/2019-making-pycon-india-women-friendly.html), from **last year - we would like**

a '-' instead of '.'

**Review by sukanyamandal — CHANGES_REQUESTED** [review]
Date of the blog - "2020-09-30"

File: content/blog/2020/diversity-inclusion-track.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+Title: Presenting our Diversity and Inclusion Track 2020!
+Date: 2020-09-14 00:00
```
Comment (palnabarun) [inline content/blog/2020/diversity-inclusion-track.md:2]:
```suggestion
Date: 2020-09-30 21:00
```

File: content/blog/2020/diversity-inclusion-track.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+Author: Sukanya Mandal
+Category: 2020
+Slug: diversity-inclusion-track
```
Comment (palnabarun) [inline content/blog/2020/diversity-inclusion-track.md:5]:
```suggestion
Slug: diversity-and-inclusion-track
```

File: content/blog/2020/diversity-inclusion-track.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+Summary: PyCon India releases its schedule and details for the 2-day diversity and inclusion track happening this year!
+
+Diversity and Inclusion have been our utmost priority. To make the conference more accessible we had taken a series of constructive steps to build an inclusive space for all. Promoting the sharing of knowledge and diverse insights by the community without inhibitions and still managing to keep 6 feet apart. 
+
+As per our [mission statement](https://in.pycon.org/blog/2019/2019-making-pycon-india-women-friendly.html), from last year. We would like to ensure that the conference approaches everyone with respect and that we resolve differences peacefully. In particular, we will proactively extend our invite to women, provide support for students, and ensure that people with disabilities feel welcome at the conference.
```
Comment (palnabarun) [inline content/blog/2020/diversity-inclusion-track.md:10]:
```suggestion
As per our [mission statement](https://in.pycon.org/blog/2019/2019-making-pycon-india-women-friendly.html), we would like to ensure that the conference approaches everyone with respect and that we resolve differences peacefully. In particular, we will proactively extend our invite to women, provide support for students, and ensure that people with disabilities feel welcome at the conference.
```

**Review by palnabarun — CHANGES_REQUESTED** [review]
Probably this requires schedule changes as well.

**Comment by vipulgupta2048** [comment]
Added all suggestions!

**Review by palnabarun — APPROVED** [review]

**Pull request merged**